### PR TITLE
FIX to account for different versions of sphinx_gallery

### PR DESCRIPTION
**Pull request by jasmainak**

### CODE REVIEW

**Comment by jasmainak** [comment]
See: http://sphinx-gallery.readthedocs.org/en/latest/getting_started.html. It's called `sphinx_gallery` now

File: doc/conf.py [hunk 1]
```diff
@@ -17,7 +17,12 @@
 import os.path as op
 from datetime import date
 
-import sphinxgallery
+try:
+    import sphinx_gallery as sg
+    extension = 'sphinx_gallery.gen_gallery'
+except:
```
Comment (Eric89GXL) [inline doc/conf.py]: `except ImportError` or at least `except Exception` (what you have now will even catch `SystemExit`)

File: doc/conf.py [hunk 2]
```diff
@@ -17,7 +17,12 @@
 import os.path as op
 from datetime import date
 
-import sphinxgallery
+try:
+    import sphinx_gallery as sg
+    extension = 'sphinx_gallery.gen_gallery'
```
Comment (agramfort) [inline doc/conf.py]: sg_extension?

**Comment by jasmainak** [comment]
comments addressed

**Comment by agramfort** [comment]
merge when travis is happy

thx

**Comment by agramfort** [comment]
thx @jasmainak

**Pull request merged**